environment.ci.yml

```diff
@@ -4,9 +4,13 @@ channels:
     - conda-forge
 dependencies:
     - codecov
-    - flake8
+    - flake8==4.0.1
     - mypy
-    - pandas
+    - pytest
     - pytest-cov
-    - scikit-learn
```

Comment:
why sklearn is removed ?

Reply:
same question for the txt env files

Reply:
Because of the dependance of mapie installed with `pip install -e`, we also import these packages.